analysis/utility.py

```diff
@@ -0,0 +1,53 @@
+#/usr/bin/python 
+# npagane | python utilities file
+
+import pathlib
+import numpy as np
+
+# set parameters
+lengthPerBP = 0.332
```

Comment:
Python programmers prefer `snake_case` for regular variable names (see [Pep8](https://www.python.org/dev/peps/pep-0008/#id37))